galaxy_ng/app/api/ui/serializers/user.py

```diff
@@ -0,0 +1,80 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from galaxy_ng.app.api import permissions
+
+from galaxy_ng.app.models import auth as auth_models
+
+
+class GroupSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = auth_models.Group
+        fields = (
+            'id',
+            'name'
+        )
+
+
+class UserSerializer(serializers.ModelSerializer):
+    password_confirm = serializers.CharField(write_only=True, allow_blank=True, required=False)
```

Comment:
While confirmation field is useful when working with HTML forms and user interface, I don't understand why it has to be a part of the API.

Reply:
Oh, that's a good point. We can just validate this on the UI.